posthog/utils.py

```diff
@@ -390,6 +391,14 @@ def render_template(
                 posthog_app_context["frontend_apps"] = get_frontend_apps(user.team.pk)
                 posthog_app_context["default_event_name"] = get_default_event_name(user.team)
 
+            if user.email:
+                # TODO: Convert to an SDK function call
+                posthog_app_context["current_user_hash"] = hmac.new(
```

Comment:
One change we should make here is to have two clear values `current_user_id_hash` and `current_user_email_hash` as otherwise it will be confusing what they are used for. The ID one would be great for "secure mode" in the future and the `email` for anything that requires emails (like this feature or maybe sending emails in the future)